integreat_cms/static/src/js/pages/machine-translation-overlay.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [0, 1, 3] }] */
+import { getCsrfToken } from "../utils/csrf-token";
+
+const limitOfPreview = 5;
```

Comment:
If the limit is a variable, I would not hardcode the number in the variable name `listOfFirstFivePages`.